### ISSUE TITLE
Group the properties and functions for Foundation Extensions

### ISSUE THREAD

**Issue by Simon-Zeng** [EFPrefix/EFFoundation#5]
#### Description

When i add both EFQRCode and SwifterSwift in cocoapods, compile error comes out. It is showing lots of "ambiguous use of 'xxx'".

#### Reproduce
Add both EFQRCode and SwifterSwift to project using cocoapods, and call methods like 'nsString' and etc.

#### Other Comment
I'd suggest to group properties and functions for Foundation Extensions like bellow:

```swift
// SwiftKit
public extension UIView {
    public class SwiftKit {
        public class func someClassMethod() {
            print("someClassMethod from Swift Kit")
        }

        public var someProperty: Double {
            print("someProperty from Swift Kit")
            return 0
        }
    }

    var SwiftKit:SwiftKit {
        return SwiftKit()
    }
}
```

**Comment by EyreFree** [comment]
@Simon-Zeng 

I found your solution on [stackoverflow](https://stackoverflow.com/questions/32736698/swift-extension-same-extension-function-in-two-modules), but it seems not work in `Array+.swift`:

```swift
import Foundation

public extension Array where Element: Equatable {

    mutating func removeFirst(_ value: Element) {
        if let index = firstIndex(of: value) {
            remove(at: index)
        }
    }

    mutating func removeAll(_ value: Element) {
        removeAll { element -> Bool in
            return value == element
        }
    }
}
```

Can you give me more advice?

**Comment by EyreFree** [comment]
@Simon-Zeng 

Fixed in [1.1.1](https://github.com/EFPrefix/EFFoundation/releases/tag/1.1.1).
